src/mame/video/zx8301.cpp

```diff
@@ -292,9 +295,14 @@ void zx8301_device::draw_line_mode8(bitmap_rgb32 &bitmap, int y, uint16_t da)
 
 			int color = (green << 2) | (red << 1) | blue;
 
+			if (flash_active) {
```

Comment:
Please be consistent with the style of the surrounding code - this file uses Allman brace indentation.

Reply:
Yeah, I noticed that after creating the pull request. Will fix.